src/scss/_functions.scss

```diff
@@ -397,3 +402,59 @@
 @function _oColorsNameExists($color-name) {
 	@return map-has-key($_o-colors-palette, $color-name);
 }
+
+/// Converts HSB/HSV values to a hex colour
+///
+/// @access private
+///
+/// @param {Number} $hue - number between 0-360
+/// @param {Number} $saturation - number between 0-100
+/// @param {Number} $brigthness - number between 0-100
+///
+/// @return {Colour} - the hex representation of given hsb values
+@function _oColorsHsbToHex($hue, $saturation, $brightness) {
+	@if $brightness == 0 {
+		@return hsl(0, 0, 0);
+	}
+
+	$hsl-luminance: ($brightness/2) * (2 - ($saturation/100));
+	$hsl-saturation: ($brightness * $saturation) / if($hsl-luminance < 50, $hsl-luminance * 2, 200 - $hsl-luminance * 2);
+	@return hsl($hue, $hsl-saturation, $hsl-luminance);
+}
+
+/// Returns HSB/HSV colour values for a given colour hex.
+///
+/// Logic derived from chroma.js
+/// https://github.com/gka/chroma.js/blob/088e18f50a3b5b1d009ce68f540265cafa0cb6a1/src/io/hsv/rgb2hsv.js#L10
```

Comment:
We need to include their license 
https://github.com/gka/chroma.js/blob/088e18f50a3b5b1d009ce68f540265cafa0cb6a1/LICENSE#L6-L17

Reply:
I wondered about that. I referenced the logic but didn't use or modify the source. The algorithm for HSL to HSB conversion is documented elsewhere.

Reply:
For example: https://www.rapidtables.com/convert/color/rgb-to-hsv.html